doc/Components.md

```diff
@@ -0,0 +1,164 @@
+# Components of CORTX
+
+## A list of the Components of CORTX
+
+1. [Motr](https://github.com/seagate/cortx-motr)
+1. [Provisioner](https://github.com/Seagate/cortx-prvsnr)
+1. [Experiments](https://github.com/Seagate/cortx-experiments)
+1. [HA](https://github.com/Seagate/cortx-ha/)
+1. [HARE](https://github.com/Seagate/cortx-hare/)
+1. [Management Portal](https://github.com/Seagate/cortx-management-portal)
+1. [Manager](https://github.com/Seagate/cortx-manager)
+1. [Monitor](https://github.com/Seagate/cortx-monitor)
+1. [POSIX](https://github.com/Seagate/cortx-posix)
+1. [S3-Server](https://github.com/Seagate/cortx-s3server)
+1. [Utils](https://github.com/Seagate/cortx-utils)
+
+## Definitions and Links to each of the components
+
+### Motr
```

Comment:
Can this word be a link itself?

Reply:
I will change it to do this but I think it looks weird when the title is blue, the only other way I have seen to make the text white while still being a link is to use HTML but I am afraid this may add complications. What do you think?